Tasks/Sergeichik_Maria/Task2.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+print(f"current time is {datetime.datetime.now().hour}:{datetime.datetime.now().minute}")
+enter_yourself = input("Введите время данным форматом hh:mm\n")
+h_m = enter_yourself .split(':')
+print(h_m)
+h = int(h_m[0])
+m = int( h_m[1])
+print(h, m)
+hours = {   1 : "один час" , 2 : "два часа" , 3 : "три часа" , 4 : "четыре часа" , 
+    5 : "пять часов" , 6: "шесть часов" , 7 : "семь часов" , 8 : "восемь часов" , 
+    9 : "девять часов" , 10 : "десять часов" , 11 : "одинадцать часов" , 12 : "двенадцать часов"}
+
+minutes = { 1 : "одна минута", 2 : "две минуты", 3 : "три минуты", 4 : "четыре минуты", 5 : "пять минут", 6 : "шесть минут", 7: "семь минут",
+8 : 'восемь минут', 9:"девять минут", 10:"десять минут", 11:"одинадцать минут", 12:"двенадцать минут", 13:"тренадцать минут",
+14:"четырнадцать минут", 15:"пятнадцать минут", 16:"шестнадцать минут", 17:"семнадцать минут", 18:"восемнадцать минут", 19:"девятнадцать минут",
+20:"двадцать минут", 21:"двадцать одна минута", 22:"двадцать две минуты", 23:"двадцать три минуты", 24:"двадцать четыре минуты", 
+25:"двадцать пять минут", 26:"двадцать шесть минут", 27:"двадцать семь минут", 28:"двадцать восемь минут", 29:"двадцать девять минут", 
+30:"половина", 31:"тридцать одна минута", 32:"тридцать две минуты", 33:" тридцать три минуты", 34:"тридцать четыре минуты", 
+35:"тридцать пять минут", 36:"тридцать шесть минут", 37:"трицать семь минут", 38:"тридцать восемь минут", 39:"тридцать девять минут", 
+40:"без двадцати минут", 41:"без девятнадцати минут", 42:"без восемнадцати минут", 43:"без семнадцати минут", 44:"без шестнадцати минут", 
+45:"без пятнадцати", 46:"без четырнадцати минут",47:"без тренадцати минут", 48:"без двенадцати минут", 49:" без одинадцати минут",
+50:"без десяти", 51:"без девяти минут", 52:"без восьми минут", 53:"без семи минут", 54:"без шести минут", 55:"без пяти минут",
+56:"без четырех минут", 57:"без трех минут", 58:"без двух минут", 59:"без одной минуты"}
+hours_1 = {1:"первого", 2:"второго", 3:"третьего", 4:" четвертого", 5:"пятого", 6:"шестого", 7:"седьмого", 8:"восьмого", 
+9:"девятого", 10:"десятого", 11:"одинадцатого", 12:"двенадцатого", 13:"первого"}
```

Comment:
the data structure is overcomplicated, but it allows building a simpler logic, which can be good